lms/static/js/commerce/track_ecommerce_events.js

```diff
@@ -24,7 +24,7 @@
                 });
             }
 
-            window.analytics.track('edx.bi.ecommerce.upsell_links_clicked', eventAttrs);
+            window.analytics.trackLink(elt, 'edx.bi.ecommerce.upsell_links_clicked', eventAttrs);
```

Comment:
I couldn't really test this because the segment analytics library isn't available to use locally (see https://github.com/edx/edx-platform/blob/master/lms/templates/widgets/segment-io.html#L4) but this appears consistent with other existing invocations of the trackLink call...